spec/features/user_edits_voter_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'user edits a voter', %(
+As a signed up user
+I want to edit a voter
+So that I can update that voter with new contact information
+) do
+  # Acceptance Criteria
+  # [ ] A user enters the voter's information
+  # [ ] If the information is valid, the voter's information is updated in the
+  #     database, and the user is taken to the voter's view page
+  # [ ] If the information is invalid, the user receives an error message, and
+  #     is returned to the edit voter page
+  scenario 'provides valid voter information' do
+    sign_in_as(FactoryGirl.create(:user))
+    voter = FactoryGirl.create(:voter)
+    visit edit_voter_path(voter)
+
+    fill_in 'First name', with: "John Q."
+    fill_in 'Last name', with: "Public"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.